desktop_version/src/FileSystemUtils.cpp

```diff
@@ -148,8 +148,7 @@ int FILESYSTEM_init(char *argvZero, char* baseDir, char *assetsPath)
 
 	if (basePath == NULL)
 	{
-		puts("Unable to get base path!");
-		return 0;
+		basePath = SDL_strdup("./");
```

Comment:
Having this as a fallback is okay but we need to have a warning message noting that we had to fall back to ".", though it's probably worth fixing this in PhysFS and SDL.

Reply:
The issue is that OpenBSD kernel doesn't support this feature see : https://openports.se/devel/sdl2